lib/libvmod_blob/base64.c

```diff
@@ -31,41 +31,176 @@
 #include "vdef.h"
 #include "vrt.h"
 #include "vas.h"
+#include "miniobj.h"
 
-#include "base64.h"
```

Comment:
IMO it's customary to include tables this large from a header file. That breaks encapsulation, but I think it's warranted for tables of numbers that now dominate the C source.